recipe/meta.yaml

```diff
@@ -61,10 +60,47 @@ outputs:
       license: BSD-3-Clause
       license_file: LICENSE
 
+  - name: clang_{{ cross_target_platform }}
+    version: {{ clang_version }}
+    script: install-clang.sh
+    build:
+      skip: True  # [win or s390x or cross_target_platform == "linux-s390x"]
+    requirements:
+      run:
+        - libgcc-devel_{{ cross_target_platform }} {{ ctng_gcc }}.*
+        # for activation of binutils env vars
+        - {{ pin_subpackage("binutils_" ~ cross_target_platform, exact=True) }}
```

Comment:
do we really need an exact pin here (aside from the fact that we're already in the same recipe and we _can_)?

This seems to be the only coupling that would _require_ building `clang_linux-64` on this feedstock.

Reply:
It's not about the coupling, but about sharing the code. It's a waste to move this to a feedstock and we have to keep these scripts in sync in yet another feedstock.